addons/hr_holidays/models/res_users.py

```diff
@@ -53,7 +53,7 @@ def _get_on_leave_ids(self, partner=False):
         self.env['hr.leave'].flush(fnames=['user_id', 'state', 'date_from', 'date_to'])
         self.env.cr.execute('''SELECT res_users.%s FROM res_users
                             JOIN hr_leave ON hr_leave.user_id = res_users.id
-                            AND state in ('validate')
+                            AND res_users.state in ('validate')
```

Comment:
Thanks for the PR!

However it should be the leave state and not the users' :-)

```suggestion
                            AND hr_leave.state in ('validate')
```

Reply:
Excellent!! it is true thank you very much